src/Microsoft.ML.Data/Data/SchemaDefinition.cs

```diff
@@ -68,30 +68,95 @@ public VectorTypeAttribute(params int[] dims)
     /// column encapsulates.
     /// </summary>
     [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
-    public sealed class ColumnAttribute : Attribute
+    public sealed class LoadColumnAttribute : Attribute
     {
-        public ColumnAttribute(string ordinal, string name = null)
+        /// <summary>
+        /// Initializes new instance of <see cref="LoadColumnAttribute"/>.
+        /// </summary>
+        /// <param name="ordinal">The index of the column in the text file.</param>
+        /// <param name="name">The optional name of the column, if it should be different from the name of the field or property where the attribute is positioned.</param>
+        /// <param name="loadAllOthers">Wheather to load all columns, besides the one specified in the ordinal.</param>
+        public LoadColumnAttribute(int ordinal, string name = null, bool loadAllOthers = false)
```

Comment:
>name [](start = 55, length = 4)

can we remove `name`? This duplicates the functionality of `ColumnName` for no good reason